tst/regression/test_suites/output_hdf5/output_hdf5.py

```diff
@@ -126,7 +126,8 @@ def Analyse(self, parameters):
             ["max", 9.43685e-01, 4.80914e-01],
             [
                 "min",
-                1.67171e-10,
+                # 1.67180e-10 if parameters.sparse_disabled else 1.67171e-10,
+                1.67180e-10,
```

Comment:
The results are now in agreement and I *think* the commented line can go, but I don't fully understand why things changed so I am hesitant to remove the old value.

Reply:
If the sparse results are now in agreement with the original "dense" results, I'd say this is an improvement and it may have fixed a hidden bug.
I only would have been concerned if the dense results would now yield the original sparse value.